036-BGP/Student/00-lab_setup.md

```diff
@@ -4,9 +4,10 @@
 
 ## Description
 
-Since this Hack is not about deploying VNets or NVAs, you can use a script to deploy the infrastructure that you will be working on. You will find a script called `bgp.sh` in the file resources supplied for this Hack, which you can run to deploy the environment. The script has been tested to run in Azure Cloud Shell. Note that the script takes around 1 hour to complete, and it requires certain dependencies to exist. Therefore, it is recommended to deploy the environment the day before the Hack:
+Since this Hack is not about deploying VNets or NVAs, you can use a script to deploy the infrastructure that you will be working on. You will find a script called `bgp.sh` in the [file resources](./Resources) supplied for this Hack, which you can run to deploy the environment. The script has been tested to run in Azure Cloud Shell. Note that the script takes around 1 hour to complete, and it requires certain dependencies to exist. Therefore, it is recommended to deploy the environment the day before the Hack:

```

Comment:
Please remove any links to the WTH repo from the student guide.  I would propose the following text to make instructions clearer for the students:

"You will find a script called `bgp.sh` in the Resources.zip file provided to you by your coach for this Hack, which you can run to deploy the environment."